### PR TITLE
Persistent zoom level

### PR DESCRIPTION
**Pull request by Studio-XP**
## Description
Keeps your last zoom level for when you reload the page

## Motivation and Context
Having a large scan radius (15 steps), the default zoom level of 16 wasn't showing me everything around me, and I would have to zoom out to 14 (13 on mobile).  Every time I would refresh the page, I would have to zoom back out again

## How Has This Been Tested?
Chrome on Desktop and Android

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by rrbailey89** [comment]
Yes please

**Comment by chuyskywalker** [comment]
Why does `storeZoom` flip back and forth? I'm not sure I understand the logic there.

**Comment by Studio-XP** [comment]
I'm not sure if that's the best way to accomplish this, but that prevents the zoom level from being stored if you click on the notification to open three map on a notified Pokemon. The moment you zoom after that it starts to store it again

**Comment by chuyskywalker** [comment]
Oooohhh, `centerMap` is only called for notifications?

**Comment by Studio-XP** [comment]
In this case, yes

**Comment by CoLdAs1cE** [comment]
oh, i was hoping this was for map in general so it would start zoomed out so i didnt have to some out every time ;(

**Comment by Studio-XP** [comment]
@CoLdAs1cE I think you misunderstood the conversation up there. What this does is store your zoom level, so if you zoom out or zoom in, when you reload the page it sets it to that level. Say you zoom out 3 levels to level 13 (default is 16) to get a larger view of the area, if you reload the page it'll load it at level 13.

What we were talking about up there was that it ignores the zoom level only if you click on the notification window that zooms you into a selected Pokemon/rarity, but picks it right back up if you zoom after that. Say you're at zoom level 13 and you click on the notification that centers and zooms you into a Pokemon (zoom level 20) and then reload the page. You don't want it to load the page zoomed in at that close of a level.

tl;dr: it works whenever you zoom, and ignores the auto-zoom from map notifications

**Comment by CoLdAs1cE** [comment]
ah i clearly miss understood :) :+1:  that's good lets get this done thank you for that clarification

**Comment by chuyskywalker** [comment]
@Studio-XP get this a rebase tonight and I'll merge it. Thanks!

**Comment by Studio-XP** [comment]
Sorry, by rebase do you mean pull it against a current version of develop? I'm still fairly new to contributing with github

**Comment by Studio-XP** [comment]
Rebased as #814
